composer.json

```diff
@@ -171,13 +170,22 @@
         "symfony-app-dir": "app",
         "symfony-bin-dir": "bin",
         "symfony-var-dir": "var",
-        "symfony-web-dir": "web",
+        "symfony-web-dir": ".",
         "symfony-tests-dir": "tests",
         "symfony-assets-install": "relative",
         "incenteev-parameters": {
             "file": "app/config/parameters.yml"
         }
     },
-    "author": "PrestaShop",
```

Comment:
for what it means, the "author" key is not allowed in Composer configuration => https://getcomposer.org/doc/04-schema.md#authors